### DISCUSSION TITLE
HammerDB - Schema Build crashes on Windows Server 2016 -

### DISCUSSION THREAD

**Discussion by venkisub** [TPC-Council/HammerDB#232] [Q&A] [answered]
Hi,

I am new to HammerDB. Did read some documents and started trying it out. I hit an error as attached on Windows when I run the Schema Build. I am trying to create 200 warehouses with 10 Vusers. The database is Oracle on Linux.

![Wish_Crashes](https://user-images.githubusercontent.com/81671517/114670967-22a58600-9d36-11eb-9912-3e4dcde35fbb.png)

**Comment by sm-shaw** [comment, marked as answer]
As a first step, did you follow the instructions in the release notes: https://www.hammerdb.com/docs/ch01s01.html?

1.4. Known Third-Party Driver Issues
1.4.1. Oracle on Windows: Oracle Bug 12733000 OCIStmtRelease crashes or hangs if called after freeing the service context handle

This is a long-standing issue with the Oracle client in a multi-threaded environment and the release notes show how it is resolved.

**Reply by venkisub** [reply]
Thank you!
I assumed it as Oracle database on Windows whereas now I understand it is for Oracle Clients.  I updated sqlnet.ora and it fixed my issues. Thank you!
